src/content/components/TimelineViewport.js

```diff
@@ -17,7 +17,9 @@ import type {
 import type { UpdateProfileSelection } from '../actions/profile-view';
 import type { ProfileSelection } from '../actions/types';
 
-const { DOM_DELTA_PAGE, DOM_DELTA_LINE } = new WheelEvent('mouse');
+const { DOM_DELTA_PAGE, DOM_DELTA_LINE } = (typeof window === 'object' && window.WheelEvent)
```

Comment:
The problem is that this runs anytime the document is imported, often before I get a chance to be able to inject something into the window, and I have to catch it for every single test that happens to touch this file.